greenpithumb/pump.py

```diff
@@ -94,16 +96,20 @@ def pump_if_needed(self, moisture):
 class PumpScheduler(object):
     """Controls when the pump is allowed to run."""
 
-    def __init__(self, local_clock, sleep_windows):
+    def __init__(self, local_clock, sleep_windows, pump_interval):
```

Comment:
This feels a bit weird because it's outside the scope of what `PumpScheduler` is defined to do. But also it feels like it fits in better with `PumpManager` because then `PumpManager` is in charge of turning the pump on, `PumpScheduler` is responsible for keeping the pump off.

I think there's a temptation to go `PumpScheduler` because it knows what time it is, but what if we created something like a `Timer` class like that we could call `timer.done()` and `timer.reset()` and gave that to the `PumpManager` so it still only needs one new param?